build.gradle

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2019 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+
+    // https://docs.gradle.org/current/userguide/java_library_plugin.html
+    id "java-library"
+
+    // https://docs.gradle.org/current/userguide/checkstyle_plugin.html
+    id "checkstyle"
+
+    // https://docs.gradle.org/current/userguide/jacoco_plugin.html
+    id "jacoco"
+
+    // https://docs.gradle.org/current/userguide/distribution_plugin.html
+    id "distribution"
+
+    // https://docs.gradle.org/current/userguide/publishing_maven.html
+    id "maven-publish"
+
+}
+
+repositories {
+    jcenter()
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+compileJava {
+    options.compilerArgs = ['-Xlint:all', '-Werror']
+}
+
+checkstyle {
+    toolVersion "8.29"
+    configDir rootProject.file("checkstyle/")
+}
+
+jacoco {
+    toolVersion = "0.8.5"
+}
+
+distributions {
+    main {
+        contents {
+            from jar
+            from configurations.runtimeClasspath
+
+            into("/") {
+                from projectDir
+                include "version.txt", "README*", "LICENSE*", "NOTICE*", "licenses/"
+                include "config/"
+            }
+        }
+    }
+}
+
+publishing {
+    publications {
+        maven(MavenPublication) {
+            // Defaults, for clarity
+            groupId = getGroup()
+            artifactId = getName()
+            version = getVersion()
+
+            pom {
+                name = "Aiven Kafka Elasticsearch connector"
+                description = "A Kafka Elasticsearch connector for copying data between Elasticsearch and Kafka."
```

Comment:
Let's put it this way: 
`"A Kafka Elasticsearch sink connector for copying data from Kafka to Elasticsearch."`